### PR TITLE
Added some new sentences and tests for EN US

### PR DESCRIPTION
**Pull request by waltlillyman**
US English speakers can be flexible in use of the article, "the", especially with voice assistants.
I've tried to include & exclude it in my sentences and tests, based on what I've heard here in the heartland of the mid-western USA, St. Louis, Missouri.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (balloob) [inline .vscode/settings.json:1]: Could you remove this file from your PR. We have a PR open to have VSCode ignore this.

**Review by balloob — APPROVED** [review]
Nice!

**Comment by waltlillyman** [comment]
Merged, so moving on.

**Comment by waltlillyman** [comment]
Reopened !

File: sentences/en/climate_HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -5,3 +5,4 @@ intents:
       - sentences:
           - "<what_is> [the] <temp>"
           - "<what_is> [the] <temp> in <area>"
+          - "<what_is> [the] <area> <temp>"
```
Comment (synesthesiam) [inline sentences/en/climate_HassClimateGetTemperature.yaml:8]: Hi @waltlillyman, this is definitely not obvious but `<area>` and `<name>` already include an optional `[the]`: https://github.com/home-assistant/intents/blob/e848342fed0328e312ca9efca4e283dd5ee8a546/sentences/en/_common.yaml#L42

**Review by synesthesiam — CHANGES_REQUESTED** [review]

File: sentences/en/climate_HassClimateGetTemperature.yaml (hunk 2 above)
Comment (synesthesiam) [reply sentences/en/climate_HassClimateGetTemperature.yaml:8]: We will also need to add at least as many test cases as there are input sentences for each intent :+1:

Comment (waltlillyman) [reply sentences/en/climate_HassClimateGetTemperature.yaml:8]: Ok, I can resume tomorrow AM, US CT!

Comment (synesthesiam) [reply sentences/en/climate_HassClimateGetTemperature.yaml:8]: Thanks! :slightly_smiling_face:

Comment (waltlillyman) [reply sentences/en/climate_HassClimateGetTemperature.yaml:8]: Made those corrections and tested locally successfully.

**Comment by waltlillyman** [comment]
@synesthesiam @balloob I removed all redundant [the] from before {name} and {area}, and ensured that each sentence had at least one corresponding test case. I ran tests locally successfully. Please take a look!

File: sentences/en/climate_HassClimateGetTemperature.yaml (hunk 2 above)
Comment (balloob) [reply sentences/en/climate_HassClimateGetTemperature.yaml:8]: This is not solved yet. We should remove `[the]` as it's included in the `<area>` expansion.

```suggestion
          - "<what_is> <area> <temp>"
```

**Pull request merged**